.github/workflows/e2e-main.yaml

```
@@ -192,6 +210,16 @@ jobs:
         exclude:
         - installation: ${{ (github.event.inputs.update_with_extensions && github.event.inputs.update_with_extensions == 'true') && 'N/A' || 'custom-extensions' }}
     steps:
+      - name: Set the default env. variables
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)